logos/tests/unit/capacity/test_demand_tracker.py

```
@@ -162,6 +162,102 @@ def test_burst_counts_in_stats():
+    l5 = tracker._load_avg["steady"]["5m"]
+    # Steady-state for _record_load_locked at 1Hz with unit u = 60/tau is geometric:
+    # L = u / (1 - exp(-1/tau)).
+    expected_1m = (60.0 / 60.0) / (1.0 - math.exp(-1.0 / 60.0))
+    expected_5m = (60.0 / 300.0) / (1.0 - math.exp(-1.0 / 300.0))
+    assert abs(l1 - expected_1m) / expected_1m < 0.01
+    assert abs(l5 - expected_5m) / expected_5m < 0.01
+
+
+def test_loadavg_decays_when_idle():
+    """Without traffic, loadavg should decay toward zero."""
+    import time
```

Flake8 (7.3.0)

[error] 'time' imported but unused

(F401)